### PR TITLE
Adding Tests for checking if all the files for labels exists

### LINKED ISSUE #144: Add a test that checks that all files mentioned in labels files actually exist

### PR DESCRIPTION
**Pull request by Shashi456**
Fixes #144.

I was confused about where to add the test so i made a file. If it needs to be shifted i will do the same. 
@marco-c

### CODE REVIEW

File: tests/test_labels.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import csv
+import os
+
+
+def test_labels():
+    with open('..\labels.csv') as csvfile:
```
Comment (marco-c) [inline tests/test_labels.py]: There's a function in utils to read labels, why do you keep rewriting the reading manually? :smile:

File: tests/test_labels.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        reader = csv.reader(csvfile, delimiter=',')
+        for row in reader:
+            if row[0] == 'Image Name':
+                continue
+            file_name_firefox = '..\data\\' + row[0] + '_' + 'firefox.png'
```
Comment (marco-c) [inline tests/test_labels.py]: Nit: Use os.path.join

File: tests/test_labels.py (hunk 1 above)
Comment (Shashi456) [reply tests/test_labels.py]: I forget about the functions :P

File: tests/test_labels.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import os
+
+from autowebcompat import utils
+
+
+def test_labels():
+    labels = utils.read_labels()
+    for label in labels:
+        if label == 'Image Name':
```
Comment (marco-c) [inline tests/test_labels.py]: The read_labels function skipts the header, so this should be unnecessary.

File: tests/test_labels.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+            continue
+        for browser in ['firefox', 'chrome']:
+            assert os.path.join('..\data\\', '{}_{}.png'.format(label, browser))
```
Comment (marco-c) [inline tests/test_labels.py]: With os.path.join there's no need to use the `\` characters.

**Pull request merged**